### PR TITLE
Portfolio : Project 8----Details modal

### PR DESCRIPTION
**Pull request by V0pe**
### Portfolio : Details modal

In this project, we add the following features.

- Create the portfolio details modal to the desktop and mobile. 
- Fix the layout of the modal overlay.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -17,3 +17,313 @@ for (let i = 0; i < 3; i++) {
     navbar.style.top = "-110vh";
   })
 }
+
+// Modal DOM
+
+const portfolioPopUp = document.createElement('div'); // Create popup div
+portfolioPopUp.className = 'popup';
+portfolioPopUp.innerHTML = `
+<div class="content">
```
Comment (codepantha) [inline script.js]: - [ ] You should have a projects array in your `script.js` file 🗄️. The projects array would be an array of objects containing all your projects and necessary details about the projects. For the avoidance of doubt, each project could be an object in the projects array. 🙏

The projects array should look something like this:
```
const projects = [
  {
    name: "My project",
    description: "My project's description",
    image: "image.png",
    link: "live_link.com",
  }
]
```

- [ ] Nice feature you implemented where the modal disappears if the user clicks outside it 👍 However, the popup modal should be populated with details from your `projects` array. Right now it looks like you are displaying the same modal each time a user clicks the 'see project' button 😎 

- [ ]  I noticed that your projects section is still hardcoded. Please refer to the `projects ` array suggestion above to dynamically display your projects. Loop through the array and use it's contents to display all your projects in the projects section. The advantage of this is that if you have a new project or you want to remove an older project, you can do it in one place only - the projects array. 🙏
https://github.com/V0pe/Portifolio/blob/f07f55d851bf7e17c60f15335bd9b912d60b94b3/index.html#L78

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @V0pe 👋, 

## STATUS: CHANGES REQUIRED ♻️ 

You have done a terrific job so far 👏 👏 !
There are some issues that you still need to work on to go to the next project but you are almost there, trust me! 👍 


### To highlight 💡 
- Clean code ✅ 
- Descriptive PR ✅ 
- Professional README ✅ 
- Beautiful UI ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (V0pe) [reply script.js]: Thanks @codepantha. I have adjusted the changes and added the dynamics.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @V0pe , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Clean code.
:heavy_check_mark: Well implemented suggested changes.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**